src/rules/getRuleComponentErrorMessage.js

```diff
@@ -12,11 +12,11 @@ governing permissions and limitations under the License.
 
 module.exports = function (ruleComponentName, ruleName, error) {
   return (
-    'Failed to execute ' +
+    'Failed to execute "' +
```

Comment:
While I love the readability of quoting the variables, do you think there are breaking change concerns with anyone reading out the errors already there?

Reply:
There may be, but I don't think console messages should be considered public API, so I'm not really worried about it. I think making them more understandable is more important.